fiftyone/operators/builtin.py

```diff
@@ -1616,43 +1639,61 @@ def config(self):
     def resolve_input(self, ctx):
         inputs = types.Object()
 
-        if ctx.dataset.media_type != fom.GROUP:
-            prop = inputs.str(
-                "msg",
-                label="This dataset does not contain groups",
-                view=types.Warning(),
-            )
-            prop.invalid = True
-        else:
-            slice_selector = types.AutocompleteView()
-            group_slices = ctx.dataset.group_slices
-            for key in group_slices:
-                slice_selector.add_choice(key, label=key)
-
-            inputs.enum(
-                "name",
-                slice_selector.values(),
-                default=ctx.group_slice,
-                required=True,
-                label="Group slice",
-                description="The group slice to delete",
-                view=slice_selector,
-            )
+        _delete_group_slice_inputs(ctx, inputs)
 
         return types.Property(
             inputs, view=types.View(label="Delete group slice")
         )
 
     def execute(self, ctx):
-        name = ctx.params["name"]
+        names = _to_string_list(ctx.params["names"])
 
-        ctx.dataset.delete_group_slice(name)
-        if ctx.group_slice == name:
+        curr_slice = False
+        for name in names:
+            curr_slice |= name == ctx.group_slice
+            ctx.dataset.delete_group_slice(name)
+
+        if curr_slice:
             ctx.ops.set_group_slice(ctx.dataset.default_group_slice)
 
         ctx.ops.reload_dataset()
 
 
+def _delete_group_slice_inputs(ctx, inputs):
+    if ctx.dataset.media_type != fom.GROUP:
+        prop = inputs.str(
+            "msg",
+            label="This dataset does not contain groups",
+            view=types.Warning(),
+        )
+        prop.invalid = True
+        return
+
+    group_slices = ctx.dataset.group_slices
+
+    names = _to_string_list(ctx.params.get("names", []))
+
+    slice_selector = types.AutocompleteView(multiple=True)
```

Comment:
Ideally can make this `DropdownView(multiple=True)` to avoid manually enforcing valid values below